src/pretix/presale/views/order.py

```diff
@@ -129,6 +129,12 @@ def position(self):
             addon_to__isnull=True,
             order__code=self.kwargs['order'],
             positionid=self.kwargs['position']
+        ).annotate(
+            checkin_count=Subquery(
+                Checkin.objects.filter(
+                    successful=True, type=Checkin.TYPE_ENTRY, position_id=OuterRef('pk')
+                ).order_by().values('position').annotate(c=Count('*')).values('c')
+            )
```

Comment:
Would it make sense to annotate the checkin_count only if `self.request.event.settings.show_checkin_number_user` is set? Not sure though if this is worth it from a performance perspective. Also, if we want to always show the checkin_count somewhere else (e.g. in pretix-control) it makes no sense to not always annotate.